src/Microsoft.ML.StaticPipe/TextLoaderStatic.cs

```diff
@@ -5,12 +5,12 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.ML.Core.Data;
-using Microsoft.ML.StaticPipe;
+using Microsoft.ML.Data;
 using Microsoft.ML.StaticPipe.Runtime;
 
-namespace Microsoft.ML.Data
+namespace Microsoft.ML.StaticPipe
 {
-    public sealed partial class TextLoader
+    public static class TextLoaderStatic
```

Comment:
>TextLoaderStatic [](start = 24, length = 16)

out of curiosity, why we have to have this suffix Static for TextLoader? It's already in different namespace.

Reply:
Couple of reasons:
1) In general I found it more convenient  to understand  which TextLoader I was using  (the Dynamic `TextLoader`  or its static counterpart `TextLoaderStatic`)
2) Wanted the class to match the filename, esp since I was moving files around.

---
In reply to: [244827020](https://github.com/dotnet/machinelearning/pull/1930#discussion_r244827020) [](ancestors = 244827020)